zigbee/XBee_Sensor_Simulator.py

```diff
@@ -146,7 +146,15 @@ def discover(self, callback):
         to refresh the status of the other XBee devices.
         """
 
-        pass
+        # The simulator does not use XBee device discovery because it does not
+        # use the actual XBee library that provides this functionality. We
+        # simulate the process by calling the callback with the packet manually.
+        for vehicle in [1, 2]:
```

Comment:
Perhaps you can make this more dynamic by using `for vehicle in range(1, self.scheduler.number_of_sensors+1)` instead. We can assume for now in the control panel view that we have 2 vehicle sensors, but we should probably solve this in a similar way later on as well.